### ISSUE TITLE
Set shape opacity on shapeBuilder doesn't work

### ISSUE THREAD

**Issue by lolegrand** [burhanrashid52/PhotoEditor#516]
**Describe the bug**
The opacity is not set when using "withShapeOpacity" on shape builder.

**To Reproduce**
1. Create a new ShapeBuilder
2. Set the opacity using withShapeOpacity method
3. set the shape on the photoEditor using the setShape method

**Expected behavior**
Set the opacity

**Smartphone Info:**
 - Device: ONE PLUS 7T

**Code sample**

```Kotlin
    override fun onClickValidateColor(color: Int, opacity: Int, brush: Int) {
        photoEditor.setBrushDrawingMode(true)

        val shapeBuilder = ShapeBuilder()
            .withShapeOpacity(opacity * 255 / 100)
            .withShapeSize(brush.toFloat())
            .withShapeColor(color)

        photoEditor.setShape(shapeBuilder)

        this.opacity = opacity
        this.brush = brush
        this.color = color
    }
```

**Comment by wasky** [comment]
Fixed by #495

**Comment by burhanrashid52** [comment]
> Duplicate of #493. Fixed by #495.

The only thing pending from my side is to publish this on maven.

**Comment by lolegrand** [comment]
Oh, sorry i hadn't noticed the issue. Good work for the lib otherwise 👍

**Comment by burhanrashid52** [comment]
Please check this in the newly Published 3.0.0 version. If it still persists then please reopen this issue.

**Comment by gabrielnakhata** [comment]
@burhanrashid52, Good Afternoon! #493 -> #495 has already been validated and accepted, but has not yet been published on maven.

**Comment by wasky** [comment]
The fix is already released in version `3.0.0`.

**Comment by gabrielnakhata** [comment]
hey @wasky  it's true i forgot to update to 3.0.0, rsrs :)
